### PR TITLE
update intro

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: fulltime-developer.md [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
 
-Mr. Henry is an internet-bureau, graphic design firm, development lab, birthplace of concepts and ideas & creative cave... We love what we do! Mr. Henry is a small team, working on both little and big projects, be it on- or offline; but all with the same vigor!
+We work closely together across teams and disciplines.  
+Always trying to improve our work, and learn from each other.
 
-Our 'clientèle' ranges from rockstars to architects, photographers to cities. Some of our clients include KOMONO, Eastpak, M HKA, Hugo & Marie, OkO, Vedett, Hi-ReS!, August, Sagmeister & Walsh, Us by Night, Kunstenpunt, YUST, our moms, city of Ghent/Antwerp/Hasselt, The Mill and [Mr. Henry themself](https://www.mrhenry.be).
+We're based in Antwerp, Belgium.
 
-Some links for some behind-the-scene action: [www](https://www.mrhenry.be) - [insta](https://instagram.com/wearemrhenry)  - [fb](https://fb.com/wearemrhenry) 
+[www](https://www.mrhenry.be) - [insta](https://instagram.com/wearemrhenry)
```
Comment (romainmenke) [inline fulltime-developer.md]: Do we still want to include social profiles here?

FB is definitely a faux pass, but maybe insta is aswel?
Maybe just the website is sufficient here?

Comment (clrblnd) [reply fulltime-developer.md]: Preference for Instagram to stay, since it shows a succinct and relevant overview of work.

Comment (fd) [reply fulltime-developer.md]: just our website is fine. also you need to have an insta account to see stuff which is a big no-no imho.

File: fulltime-developer.md [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
-Based on new, modern, and future-proof web standards, we're trying to build websites that work, everywhere, for everyone. For every *Shiny New Thing™*, we want to raise a thing that *Just Works™*.
+## Who we are
 
-If this tickles your curiosity, we'd like to talk to you.
-If you think we're 100% in the wrong, we'd still like to talk to you.
+Mr. Henry is an agency with a small team of passionate experts.  
+We're a mix of developers and designers working on online and offline projects.  
 
-## Who we are
+We work on branding, design and web development.
```
Comment (clrblnd) [inline fulltime-developer.md]:
```suggestion
We work on design, strategy and web development.  
```

Terms taken from https://www.mrhenry.be/work/

File: fulltime-developer.md [hunk 3]
```diff
@@ -1,39 +1,37 @@
...
+## Who we are
 
-If this tickles your curiosity, we'd like to talk to you.
-If you think we're 100% in the wrong, we'd still like to talk to you.
+Mr. Henry is an agency with a small team of passionate experts.  
+We're a mix of developers and designers working on online and offline projects.  
 
-## Who we are
+We work on branding, design and web development.  
+For many projects we do the full package, from branding to print and web deliverables.
```
Comment (clrblnd) [inline fulltime-developer.md:13]:
```suggestion
For many projects we do the full package, from branding to print and web deliverables. With this offer, we foster longterm relationships with our clients.
```

Emphasise the longterm

File: fulltime-developer.md [hunk 4]
```diff
@@ -1,39 +1,37 @@
...
 
-Mr. Henry is an internet-bureau, graphic design firm, development lab, birthplace of concepts and ideas & creative cave... We love what we do! Mr. Henry is a small team, working on both little and big projects, be it on- or offline; but all with the same vigor!
+We work closely together across teams and disciplines.  
+Always trying to improve our work, and learn from each other.
```
Comment (clrblnd) [inline fulltime-developer.md]:
```suggestion
We work closely together across teams and disciplines. Always trying to improve our work, and learn from each other.
```

File: fulltime-developer.md [hunk 5]
```diff
@@ -1,39 +1,37 @@
...
 
-Our 'clientèle' ranges from rockstars to architects, photographers to cities. Some of our clients include KOMONO, Eastpak, M HKA, Hugo & Marie, OkO, Vedett, Hi-ReS!, August, Sagmeister & Walsh, Us by Night, Kunstenpunt, YUST, our moms, city of Ghent/Antwerp/Hasselt, The Mill and [Mr. Henry themself](https://www.mrhenry.be).
+We're based in Antwerp, Belgium.
```
Comment (clrblnd) [inline fulltime-developer.md]:
```suggestion
Our work takes us across various industries such as government, cultural organisations, non-profits, hospitality, architecture, academia and more, both national and international.

We're based in Antwerp, Belgium.
```

Interesting to showcase we're not pigeonholed into a certain type of client. Not sure yet about exact phrasing.

**Review by clrblnd — COMMENTED** [review]
some notes

File: fulltime-developer.md (hunk 5 above)
Comment (romainmenke) [reply fulltime-developer.md]: I prefer to let our work speak for itself in this regard.

Given that the job posting is for a developer it is also less important to stress this aspect of Mr. Henry. For a creative profile I would agree that this is very important to explain well.

File: fulltime-developer.md (hunk 2 above)
Comment (romainmenke) [reply fulltime-developer.md]: I strongly prefer `web development` over `technology`.
Given that we do only web development and nothing beyond that :D

File: fulltime-developer.md (hunk 3 above)
Comment (romainmenke) [reply fulltime-developer.md:13]: How about :

```
We foster long term and sustainable relationships with our clients.
```

But as its own paragraph, not after `... and web deliverables.`

Comment (clrblnd) [reply fulltime-developer.md:13]: ok

File: fulltime-developer.md (hunk 1 above)
Comment (romainmenke) [reply fulltime-developer.md]: We definitely need to reword `insta` to `instagram`.
`insta` is just too cringe :)

----

My main motivation to remove these social profiles, aside from their possible obsolescence, is that they are not representative of our development work.

But I can understand that they show a glimpse of Mr. Henry in general, which might be interesting to candidates.

File: fulltime-developer.md [hunk 6]
```diff
@@ -92,5 +91,3 @@ _We are looking for someone who can do enough to be productive and is willing to
 We offer a full-time position in a creative environment where you can learn from and with your colleagues.
 
 [Apply here](mailto:jobs@mrhenry.be)
-
-<img src="http://from.mrhenry.be/withlove/thanks-padding.png">
```
Comment (romainmenke) [inline fulltime-developer.md:96]: Only removing this because it is broken.
I don't have a strong opinion about including a trailing image :)

Comment (clrblnd) [reply fulltime-developer.md:96]: it can go

File: fulltime-developer.md (hunk 5 above)
Comment (clrblnd) [reply fulltime-developer.md]: Ok

**Review by clrblnd — APPROVED** [review]

File: fulltime-developer.md [hunk 7]
```diff
@@ -1,39 +1,37 @@
...
-If you think we're 100% in the wrong, we'd still like to talk to you.
+Mr. Henry is an agency with a small team of passionate experts.  
+We're a mix of developers and designers working on online and offline projects.  
 
-## Who we are
+We work on design, strategy and web development.  
+For many projects we do the full package, from branding to print and web deliverables.
 
-Mr. Henry is an internet-bureau, graphic design firm, development lab, birthplace of concepts and ideas & creative cave... We love what we do! Mr. Henry is a small team, working on both little and big projects, be it on- or offline; but all with the same vigor!
+We foster long term and sustainable relationships with our clients.
```
Comment (fd) [inline fulltime-developer.md]:
```suggestion
We foster long-term and sustainable relationships with our clients.
```

**Review by fd — APPROVED** [review]

**Comment by romainmenke** [comment]
Thank you all for the feedback on this!

**Pull request merged**